mklive.sh.in

```diff
@@ -316,8 +301,13 @@ while getopts "a:b:r:c:C:T:Kk:l:i:I:S:s:o:p:v:h" opt; do
     esac
 done
 shift $((OPTIND - 1))
+
+# Check for the existance of required executables in the host system.
+check_tools
+
 XBPS_REPOSITORY="$XBPS_REPOSITORY --repository=https://repo-default.voidlinux.org/current --repository=https://repo-default.voidlinux.org/current/musl"
 
+# mklive only supports x86_64 and i686 with glibc or musl, abort on other archs
```

Comment:
I am not sure if this comment is really needed, I think the code is pretty clear about this.